news/gh-11040-gitpod-codespaces

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* Enable one-click gitpod and GitHub Codespaces setup for Linux development.
```

Comment:
We should maybe add here  a `(preview)` tag or something because we haven't used it extensively yet.